src/content/app/tools/blast/views/blast-submission-results/BlastSubmissionResults.scss

```diff
@@ -0,0 +1,38 @@
+@import 'src/styles/common';
+
+.blastSubmissionResultsContainer {
+  max-width: 1860px;
+  padding: 0 30px $global-padding-bottom 60px;
```

Comment:
the 30 and the 60 look suspiciously like the numbers that we had a whole dedicated ticket for 😃

Reply:
yes the left and right padding. I thought we can deal it in that design review ticket?